.travis-ci.sh

```diff
@@ -116,8 +115,8 @@ case "$TARGET" in
     if [ $WITH_OPAM -eq 1 ] ; then
       cat $RUNTEST_NO_DEPS;
       _boot/install/default/bin/dune runtest && \
-      _boot/install/default/bin/dune build @test/blackbox-tests/runtest-js && \
-      _boot/install/default/bin/dune build @test/blackbox-tests/runtest-coq && \
+      # _boot/install/default/bin/dune build @test/blackbox-tests/runtest-js && \
+      # _boot/install/default/bin/dune build @test/blackbox-tests/runtest-coq && \
```

Comment:
I didn't notice this; as of today the whole coq test suite is broken, including a critical regression in #3191 

What should we do?